### PR TITLE
dxScheduler: currentDateTime indicator implementation

### PR DESCRIPTION
**Pull request by Yarovaya**

### CODE REVIEW

File: styles/widgets/base/scheduler.less [hunk 1]
```diff
@@ -442,6 +445,88 @@
     display: none;
 }
 
+.dx-scheduler-header-panel-cell.dx-scheduler-header-panel-current-time-cell {
+    box-shadow: inset 0px -3px 0px 0px @SCHEDULER_APPOINTMENT_BASE_COLOR;
```
Comment (SergeyAlexeev) [inline styles/widgets/base/scheduler.less]: Should we use cross-browser mixin(`.box-shadow(...)`) here?

File: testing/tests/DevExpress.ui.widgets.scheduler/common.tests.js [hunk 2]
```diff
@@ -1039,6 +1041,57 @@ QUnit.testStart(function() {
...
+        assert.equal(this.instance.option("showCurrentTimeIndicator"), true, "showCurrentTimeIndicator option value is right on init");
+    });
+
+    QUnit.test("showCurrentTimeIndicator option should be passed to workSpace", function(assert) {
+        this.createInstance({
+            currentView: "week",
+            showCurrentTimeIndicator: false
+        });
+
+        var workSpaceWeek = this.instance.$element().find(".dx-scheduler-work-space").dxSchedulerWorkSpaceWeek("instance");
```
Comment (SergeyAlexeev) [inline testing/tests/DevExpress.ui.widgets.scheduler/common.tests.js:1056]: `var workSpaceWeek = this.instance.getWorkSpace()` ?

File: testing/tests/DevExpress.ui.widgets.scheduler/currentTimeIndicator.tests.js [hunk 3]
```diff
@@ -0,0 +1,554 @@
...
+        });
+
+        var $element = this.instance.$element();
+        assert.equal($element.find(".dx-scheduler-date-time-indicator").length, 1, "Indicator is rendered");
+        assert.equal($element.find(".dx-scheduler-date-time-indicator").length, 1, "AllDay indicator is rendered");
+
+        this.instance.option("currentDate", new Date(2017, 8, 15));
+
+        assert.equal($element.find(".dx-scheduler-date-time-indicator").length, 0, "Indicator wasn't rendered");
+        assert.equal($element.find(".dx-scheduler-date-time-indicator").length, 0, "AllDay indicator wasn't rendered");
```
Comment (SergeyAlexeev) [inline testing/tests/DevExpress.ui.widgets.scheduler/currentTimeIndicator.tests.js]: The same element `$element.find(".dx-scheduler-date-time-indicator")` is used in both asserts.

File: styles/widgets/base/scheduler.less [hunk 4]
```diff
@@ -442,6 +445,88 @@
...
+
+.dx-scheduler-date-time-indicator-line {
+        background-color:  @SCHEDULER_APPOINTMENT_BASE_COLOR;
+        position: absolute;
+        box-shadow: @SCHEDULER_TIME_INDICATOR_SHADOW;
+
+        &:before {
+            font-size: @SCHEDULER_TIME_INDICATOR_FONT_SIZE;
+            color: @SCHEDULER_APPOINTMENT_BASE_COLOR;
+            text-shadow: white 1px -1px 3px;
```
Comment (SergeyAlexeev) [inline styles/widgets/base/scheduler.less:469]: Hardcoded color `white`.

File: styles/widgets/generic/color-schemes/light/generic.light.less [hunk 5]
```diff
@@ -1844,6 +1844,7 @@
 @scheduler-recurence-editor-border-color: darken(@base-element-bg, 5%);
 @SCHEDULER_OTHER_MONTH_CELL_OPACITY: .5;
 @SCHEDULER_FILL_FOCUSED_APPOINTMENT: false;
+@SCHEDULER_TIME_INDICATOR_BACKGROUND_COLOR: lighten(@SCHEDULER_BASE_BORDER_COLOR, 9%);
```
Comment (vconst) [inline styles/widgets/generic/color-schemes/light/generic.light.less]: This constant should be editable in ThemeBuilder. Also, the name should be changed.

File: styles/widgets/base/scheduler.less [hunk 6]
```diff
@@ -442,6 +445,104 @@
...
+}
+
+.dx-scheduler-date-time-indicator-line {
+        background-color:  @SCHEDULER_APPOINTMENT_BASE_COLOR;
+        position: absolute;
+        box-shadow: @SCHEDULER_TIME_INDICATOR_SHADOW;
+
+        &:before {
+            font-size: @SCHEDULER_TIME_INDICATOR_FONT_SIZE;
+            color: @SCHEDULER_APPOINTMENT_BASE_COLOR;
```
Comment (vconst) [inline styles/widgets/base/scheduler.less:468]: Why indicator line has font-size and color settings? Currently, It does not contain any text.

**Review by vconst — CHANGES_REQUESTED** [review]

File: styles/widgets/base/scheduler.less (hunk 6 above)
Comment (Yarovaya) [reply styles/widgets/base/scheduler.less:468]: But it has a spin-icon

**Pull request closed without merging**